client/globals.d.ts

```diff
@@ -188,6 +185,9 @@ declare global {
 		adminUrl: string;
 		countries: Record< string, string >;
 		homeUrl: string;
+		locale: {
+			userLocale: string;
+		};
```

Comment:
💭 Since it's present in the `wcSettings` global browser variable, it can't hurt to let future devs know that it exists.

Another gift to future devs could be explanatory descriptions of what these are. 🎁
```suggestion
		locale: {
			/**
			 * The locale of the current user profile.
			 *
			 * @example 'en_AU' // English (Australia)
			 */
			userLocale: string;
			/**
			 * The locale of the current site, as set in WP → Settings → General.
			 *
			 * @example 'nl_NL' // Dutch (Netherlands)
			 */
			siteLocale: string;
		};
```

Reply:
That's a great suggestion! I have added it here [319c867](https://github.com/Automattic/woocommerce-payments/pull/10003/commits/319c86795f29af909665014c7ed5f312f78df7a4)